packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -432,6 +439,8 @@ export async function activate(context: vscode.ExtensionContext) {
 export function deactivate(): Promise<void> {
   console.log('SFDX CLI Extension Deactivated');
 
+  // Send metric data.
+  telemetryService.sendExtensionDeactivationEvent();
```

Comment:
I've seen some issues gathering this specific metric and found this issue on vscode (https://github.com/Microsoft/vscode/issues/47881).

Reply:
OK. It's probably not a deal-breaker if we can't get the deactivation command. I don't think important things happen in there for our extensions.